hedera-mirror-importer/src/main/resources/db/migration/v1/V1.36.1__revised_scheduled_transaction.sql

```diff
@@ -0,0 +1,27 @@
+-------------------
+-- Support revised scheduled transactions design
+-------------------
+
+-- New / missing transaction result codes
+delete from t_transaction_results where proto_id >= 207;
```

Comment:
We should narrow the scope here 
```suggestion
delete from t_transaction_results where proto_id >= 207 and proto_id <= 210;
```
or
```suggestion
delete from t_transaction_results where proto_id >= 207 and proto_id <= 214;
```

Reply:
not necessary since we know from protobuf there is no proto_id after 210

Reply:
Okay